spec/brcobranca/retorno/cnab240/base_spec.rb

```diff
@@ -0,0 +1,23 @@
+# -*_ encoding: utf-8 -*-
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.